addons/vagrant/playbooks/get_logs.yml

```diff
@@ -20,7 +20,6 @@
       shell: '{{ venom_dir }}/utils/sanitize-venom-logs.sh'
       args:
         executable: '/bin/bash'
-      no_log: True
```

Comment:
Need to bring it back.
We used that in case of logs became visible again.
Should be removed only for testing locally not in CI.